components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -363,6 +363,41 @@ void RewardsServiceImpl::Init(
   if (profile_->GetPrefs()->GetBoolean(prefs::kEnabled)) {
     StartLedger();
   }
+
+  PreferenceObserver();
```

Comment:
Maybe `InitPrefChangeRegistrar` would be a better name for this?

Reply:
fixed